packages/@lwc/template-compiler/src/__tests__/fixtures/parsing-errors/stranded-open-svg-ellipse/expected.js

```diff
@@ -1,18 +1,8 @@
-import { registerTemplate } from "lwc";
-const stc0 = {
-  attrs: {
-    xmlns: "http://www.w3.org/2000/svg",
-  },
-  key: 0,
-  svg: true,
-};
-const stc1 = {
-  key: 1,
-  svg: true,
-};
+import { parseFragment, registerTemplate } from "lwc";
```

Comment:
~~This PR is enabling static content optimization for SVG. Can you please update the PR description to include that info.~~
nvm, I read the comment about lowercasing scoped token related to svg.